styles/widgets/material/lookup.material.less

```diff
@@ -106,11 +108,6 @@
         }
     }
 
-    .dx-list-item-content {
```

Comment:
🤔

Reply:
It is a redundant style that causes wrong paddings in a grouped list inside lookup.